project/Dependencies.scala

```diff
@@ -59,10 +62,12 @@ object Dependencies {
   val sam: ModuleID = "org.broadinstitute.dsde.sam-client" %% "sam" % samV
 
   val slick: ModuleID =     "com.typesafe.slick" %% "slick"                 % slickV
-  val hikariCP: ModuleID =  "com.typesafe.slick" %% "slick-hikaricp"        % slickV
-  val mysql: ModuleID =     "mysql"               % "mysql-connector-java"  % "8.0.11"
+  val dbcp2: ModuleID = "org.apache.commons" % "commons-dbcp2" % dbcpV
```

Comment:
Is dbcp a connection pooling library? Did we confirm that this is OK in Google App Engine where threading is more controlled?

Reply:
Yes, DBCP is a connection pooling library. I had better luck with it in my simple experiments than with HikariCP. We should do some more stress testing to be sure, but there are reports of others successfully using DBCP on app engine standard.

Reply:
I also need to check in with our GCP support team. They were supposed to be getting back to me with more specific advice.